e2e-tests/native/test/pageobjects/dashboard.page.ts

```diff
@@ -12,6 +13,21 @@ class DashboardPage extends Page {
   get headerOverflowButton() {
     return $('~More options')
   }
+
+  get languageIcon() {
+    return $('~Change language')
+  }
+
+  get headerBack() {
+    return $('~Go back')
+  }
+
+  public async switchLanguageTo(language: string) {
```

Comment:
I think this should not be in the `.page`. Would make more sense to have it in the test.

Reply:
Fine for me also.
The thought behind this is to keep the tests as readable as possible and leave the implementation details inside the page object

So we could reuse the switchToLanguage also if we e.g. want to test a "date" component that should format the date for different languages (only example, no idea if we have that)